src/main/java/mate/jdbc/dao/ManufacturerDaoImpl.java

```diff
@@ -0,0 +1,125 @@
+    private static final int IS_DELETED_INDEX = 3;
+
+    @Override
+    public Manufacturer create(Manufacturer manufacturer) {
+        String insertManufacturerRequest =
+                "INSERT INTO manufacturers (name , country, is_deleted) VALUES (?,?,?)";
+        try (Connection connection = ConnectionUtil.getConnection();
+                PreparedStatement createManufacturerStatement = connection
+                           .prepareStatement(insertManufacturerRequest,
+                                   Statement.RETURN_GENERATED_KEYS)) {
+            createManufacturerStatement.setString(NAME_INDEX, manufacturer.getName());
+            createManufacturerStatement.setString(COUNTRY_INDEX, manufacturer.getCountry());
+            createManufacturerStatement.setBoolean(IS_DELETED_INDEX, false);
+            createManufacturerStatement.executeUpdate();
+            ResultSet generatedKeys = createManufacturerStatement.getGeneratedKeys();
+            if (generatedKeys.next()) {
+                Long id = generatedKeys.getObject(ID_INDEX, Long.class);
+                manufacturer.setId(id);
+            }
+
+        } catch (SQLException e) {
+            throw new DataProcessingException("Can`t insert manufacturer " + manufacturer, e);
+        }
+        return manufacturer;
+    }
+
+    @Override
+    public Optional<Manufacturer> get(Long id) {
+        String deleteRequest = "SELECT * FROM manufacturers WHERE id = ? AND is_deleted = false";
+        try (Connection connection = ConnectionUtil.getConnection();
+                PreparedStatement getManufacturerStatement =
+                        connection.prepareStatement(deleteRequest)) {
+            getManufacturerStatement.setLong(ID_INDEX, id);
+            ResultSet resultSet = getManufacturerStatement.executeQuery();
+            if (resultSet.next()) {
+                return Optional.of(createManufacturerObj(resultSet));
+            }
+            return Optional.empty();
+        } catch (SQLException e) {
+            throw new DataProcessingException("Can`t get manufacturer by id " + id, e);
+        }
+    }
+
+    @Override
+    public List<Manufacturer> getAll() {
+        String getAllRequest = "SELECT * FROM manufacturers WHERE is_deleted = false;";
+        List<Manufacturer> manufacturerList = new ArrayList<>();
+        try (Connection connection = ConnectionUtil.getConnection();
+                PreparedStatement getAllManufacturerStatement =
+                        connection.prepareStatement(getAllRequest)) {
+            ResultSet resultSet = getAllManufacturerStatement.executeQuery();
+            while (resultSet.next()) {
+                manufacturerList.add(createManufacturerObj(resultSet));
+            }
+        } catch (SQLException e) {
+            throw new DataProcessingException("Can`t get all manufacturers from DB", e);
+        }
+        return manufacturerList;
+    }
+
+    private Manufacturer createManufacturerObj(ResultSet resultSet) {
```

Comment:
private methods should follow after public as helper methods
```suggestion
    private Manufacturer getManufacturer(ResultSet resultSet) {
```